pkg/storage/cache/cache.go

```diff
@@ -71,11 +76,18 @@ func (cache *Cache) saveToDisk(key string, val interface{}) {
 }
 
 func (cache *Cache) Flush() {
+	// evict all the items in cache
 	cache.lfu.Evict(cache.lfu.Len())
+
 	close(cache.lfu.EvictionChannel)
+	// wait until cache flushing is finished
 	<-cache.cleanupDone
 }
 
+func (cache *Cache) Evit(percent float64) {
+	cache.lfu.Evict(cache.lfu.Len() / int(percent*100))
```

Comment:
```suggestion
	cache.lfu.Evict(int(cache.lfu.Len() * percent))
```
Current version works for `percent == 0.10` by an accident, it will not work for any other value :P

Reply:
right, thank u very much

Reply:
cache.lfu.Evict(int(float64(cache.lfu.Len()) * percent))